assets/src/components/Parallax/setupParallax.js

```diff
@@ -14,8 +14,6 @@ export const setupParallax = () => {
   });
 
   if (mobileSpeedAllSetup) {
-    const rellax = new Rellax('.is-style-parallax img', { // eslint-disable-line no-undef, no-unused-vars
-      center: true,
-    });
+    return new Rellax('.is-style-parallax img', {});
```

Comment:
I think that we still need the `// eslint-disable-line no-undef, no-unused-vars` line:

<img width="435" alt="Screenshot 2023-08-29 at 09 30 45" src="https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/assets/6949075/182a546e-6b53-4ea1-98cc-47e5641133b8">

Reply:
Gotcha! But I think `eslint-disable-line no-undef` is enough, since we're not defining a variable